src/styles/Mixins.scss

```diff
@@ -0,0 +1,60 @@
+@import './Colors.scss';
+
+@mixin flexCenter {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+@mixin flexColumn {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+@mixin flexAlignItemsCenter {
```

Comment:
@mixin flexAlignItemsCenter {
  display: flex;
  align-items: center;
}

@mixin ColumnJustifyContentCenter {
  justify-content: center;
  flex-direction: column;
}

@mixin FlexJustifyContentSpaceAround {
  display: flex;
  justify-content: space-around;
}

it too specific and that's why names looks bad and it's the reason why flexCenter looks good